### ISSUE TITLE
Premium capture

### ISSUE THREAD

**Issue by mbcat4333** [5P4RUX/Crunchyroll-Checker#1]
Is it possible to make it check if the account have premium?

**Comment by 5P4RUX** [comment]
Yes, capture can be added while checking the account.

13 Kas 2024 Çar 17:16 tarihinde mbcat4333 ***@***.***> şunu
yazdı:

**Comment by 5P4RUX** [comment]
> Is it possible to make it check if the account have premium?

Yes, It is possible to check whether the account is a free plan or a premium plan.
